server/apps/main/templates/main/confirmation_page.html

```diff
@@ -0,0 +1,322 @@
+              confirmation and your experience will then be published on our website. Only signed-in users who have
+              told us they are autistic will be shown your experience.
+            </li>
+
+            <hr>
+
+            <!-- Third Item Text -->
+            <li class="text">
+              <span class="asterik">*</span> If you chose to share it with researchers, it will be saved into a
+              database, and researchers who follow
+              our values will be able to use it for their research.
+            </li>
+          </ul>
+        </div>
+      </div>
+      <!-- What Happens Now Section Ends Here -->
+
+
+      <!-- What Would You Like To Do Now Section Starts Here -->
+      <div class="to_do_now">
+        <p class="h4 text-success">
+          What would you like to do now?
+        </p>
+
+        <div class="btn_flex_one mt-5">
+          <button class="btn btn-success">
+            <a href="{% url "main:upload" %}">
+              Enter Another Experience
+            </a>
+          </button>
+
+          <button class="btn btn-success">
+            <a href="{% url "main:my_stories" %}">
+              Go To My Experience Page
+            </a>
+          </button>
+        </div>
+
+        <br>
+
+        <div class="btn_flex_two">
+          <button class="btn btn-success">
+            <a href="{% url "main:public_experiences" %}">
+              View Other Experiences
+            </a>
+          </button>
+
+          <button class="btn btn-success">
+            <a href="{% url "main:index" %}">
+              Find Out More About AutSPACEs
+            </a>
+          </button>
+        </div>
+      </div>
+      <!-- What Would You Like To Do Now Section Ends Here -->
+
+    </section>
+    <!-- Confirmation Section Ends Here -->
+
+  </div>
+  <!-- Container Ends Here -->
+
+
+
+  <!-- Optional JavaScript -->
+  <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+    integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous">
+  </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
+    integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous">
+  </script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
+    integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous">
+  </script>
+
+  <!-- Custom Scripts Goes Here -->
+  <script defer>
+    const whatNow = document.getElementById("whatNow");
+    const whatNowTexts = document.getElementById("whatNowTexts");
+
+    whatNow.addEventListener("click", () => {
+      whatNowTexts.classList.toggle("d-none");
+    });
+
+  </script>
```

Comment:
Very interesting... does this custom script work for you?

Reply:
> Very interesting... does this custom script work for you?

Yes, it is, why though? Is the script getting blocked?

Reply:
I don't think it's getting blocked, it's just not working for me. But since you removed it, we're good to go!